src/gateway/kong-manager/consumer-groups.md

```diff
@@ -0,0 +1,96 @@
+* A "gold tier" with 1000 requests per minute
+* A "silver tier" with 10 requests per second
+* A "bronze tier" with 6 requests per 30 seconds
+
+The `consumer_groups` endpoint works together with the [Rate Limiting Advanced plugin](/hub/kong-inc/rate-limiting-advanced).
+
+Consumers that are not in a consumer group default to the Rate Limiting advanced
+plugin’s configuration, so you can define tier groups for some users and
+have a default behavior for consumers without groups.
+
+To use consumer groups for rate limiting, you need to:
+* Create one or more consumer groups
+* Create consumers
+* Assign consumers to groups
+* Configure the Rate Limiting Advanced plugin with the `enforce_consumer_groups`
+and `consumer_groups` parameters, setting up the list of consumer groups that
+the plugin accepts
+* Configure rate limiting for each consumer group, overriding the plugin's
+configuration
+
+For all possible requests, see the
+[Consumer Groups reference](/gateway/{{page.kong_version}}/admin-api/consumer-groups/reference).
+
+## Set up a consumer group with consumers
+
+In this section, you will create the Bronze tier consumer group with a consumer assigned to the group.
+
+1. Open the **default** workspace.
+1. From the menu, open **Consumers**, then click the **Groups** tab.
+1. Click **New Consumer Groups**.
+1. Enter `Bronze` for the consumer group name and click **Create**.
+1. From the menu, click **Consumers**, then click **New Consumer**.
+1. Enter a **Username** and **Custom ID**. For this example, you can use `Amal` for each field.
+1. Click **Create**.
+1. From the menu, click the **Groups** tab.
+1. Click the `Bronze` consumer group you just created.
+1. Click **Consumers** and **Add consumers** to add the `Amal` consumer you created to the `Bronze` consumer group.
+
+## Configure the Rate Limiting Advanced plugin for all consumers
+
+In this section, you will configure the Rate Limiting Advanced plugin to set the rate limit to 5 requests every 30 seconds for all consumers.
+
+1. Open the **default** workspace.
+1. From the menu, open **Plugins**, then click **Install Plugin**.
+1. Find the **Rate Limiting** plugin, then click **Enable**.
+1. Apply the plugin as **Global**, which means the rate limiting applies to all requests, including every service and route in the workspace.
+
+    If you switched it to **Scoped**, the rate limiting would apply the plugin to only one service, route, or consumer.
+
+    By default, the plugin is automatically enabled when the form is submitted.
+    You can also toggle the **This plugin is Enabled** button to configure the plugin without enabling it.
+    For this example, keep the plugin enabled.
+1. Complete only the following fields with the following parameters.
+    1. data config.limit: `5`
+    1. data config.window_size: `30`
+    1. data config.window_type: `sliding`
+    1. data config.retry_after_jitter_max: `0`
+    1. data config.enforce_consumer_groups: `true` 
```

Comment:
```suggestion
    1. config.limit: `5`
    1. config.window_size: `30`
    1. config.window_type: `sliding`
    1. config.retry_after_jitter_max: `0`
    1. config.enforce_consumer_groups: `true` 
    1. config.consumer_groups: `Bronze`
```
Just tested, it won't let you create the plugin without setting `consumer_groups`